OracleCloudInfrastructure/oci-cli/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM oraclelinux:8
```

```suggestion
FROM ghcr.io/oracle/oraclelinux8-python:3.6
```

This is to ensure the image is updated automatically by our CI/CD system to use the latest Oracle Linux and Python packages.